### ISSUE TITLE
[sp.pl.ligrec] Receptor/Ligand significance (dot vs torus) is not shown in the legend

### ISSUE THREAD

**Issue by gokceneraslan** [theislab/squidpy#370]
Hi guys,

I have just tried the receptor/ligand implementation in squidpy and I think it's very cool (and fast)! One little issue is that the torus/dot distinction is not really intuitive in the plots:

![image](https://user-images.githubusercontent.com/1140359/122460230-dff36a80-cf7f-11eb-9c4b-2603a5306b01.png)

Honestly, I initially thought that tori are non-significant 🙈 . I think it should have a legend like `dot: non-significant torus: significant`.

Alternatively, I think it'd make sense to show dots for all interactions and use the outline color of the dots to specify significance, e.g. from our covid paper (https://www.nature.com/articles/s41591-020-01227-z/figures/1):

<img width="1134" alt="image" src="https://user-images.githubusercontent.com/1140359/122459979-9276fd80-cf7f-11eb-903d-b37539a8a80d.png">

Main difference is that this representation allows significant but small dots whereas dot/torus does not (I mean very difficult). 

Let me know what you think and thanks for the super nice implementation 👏  😄

**Comment by gokceneraslan** [comment]
Another enhancement would be to add an option to show only the rows/cols with significant interactions (a bit different than `remove_empty_interactions` maybe something like `remove_nonsig_interactions`)

**Comment by michalk8** [comment]
Hi @gokceneraslan ,

thanks for the suggrestions, I think they're great. Re:

- torus/significance - I think it's a good idea to add more explanation to the legend and use the outline to mark significance 
- `remove_nonsig_interactions` - will also add this.

**Comment by gokceneraslan** [comment]
Thanks a lot!

**Comment by michalk8** [comment]
Hi @gokceneraslan ,

after discussing with Giovanni, we've decided to keep the tori for significance instead of colors (should be fine as long as `alpha` is a reasonable value; tried the code below with `alpha=1` and all tori were rel. visible, even for the smallest dots).
`remove_nonsig_interactions` is done.

```python
import squidpy as sq

adata = sq.datasets.visium_fluo_adata()
sq.gr.ligrec(adata, 'leiden', interactions=adata.var_names[:10])
sq.pl.ligrec(adata, 'leiden', remove_nonsig_interactions=True,
             source_groups=['0'], alpha=0.001, swap_axes=True)
```
![ligrec](https://user-images.githubusercontent.com/46717574/123556973-bc999e00-d77d-11eb-9208-b014e4e6870c.png)

Let me know if you have any further comments/suggestions.

**Comment by michalk8** [comment]
closed via #374
